mozilla-release/mobile/android/base/java/org/mozilla/gecko/tabs/TabsLayout.java

```diff
@@ -29,7 +30,15 @@
 
     private static final String LOGTAG = "Gecko" + TabsLayout.class.getSimpleName();
 
-    private final boolean isPrivate;
+    /* Cliqz Start */
+    private static final int[] STATE_PRIVATE_MODE = { R.attr.state_private };
+    private static final int[] LIGHT_THEME = { R.attr.light_theme };
+    private static final int[] LIGHT_THEME_PRIVATE_MODE = { R.attr.light_theme, R.attr.state_private };
+
+    private boolean isLightTheme;
```

Comment:
mIsLigh.....
mIsPriva.....

Reply:
did not do it

Reply:
`mIsLightTheme`. Are we planning to add a boolean for every other theme we implement or are we using numeric coding?